libs/rudder-integration-clevertap-react-native/rudder-integration-clevertap-react-native.podspec

```diff
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency "React"
-  s.dependency "Rudder-CleverTap", "~> 1.0"
+  s.dependency "Rudder-CleverTap", ">= 1.0.4"
```

Comment:
Let's provide the flexibility till minor version itself

Reply:
Addressed and implemented the same constraint in iOS.